cypress/support/constants/constants.js

```diff
@@ -124,7 +125,7 @@ module.exports = {
   EXPECTED_JSON_IN_VALIDATION_OBJECTS:
     'Expected JSON data should be defined in fixtures/objects/validationObjects/',
   EXPECTING_ERROR: 'expectingError',
-  EXTERNAL_DEVICES_PATH: 'cypress/fixtures/external/devices/',
+  EXTERNAL_DEVICES_PATH: './node_modules/configModule/testData/devices/',
```

Comment:
why did this change from external?

Reply:
retrieving the data from the node module for device data